jwt-auth.php

```diff
@@ -0,0 +1,72 @@
+<?php
+/**
+ * JWT Auth
+ *
+ * @package      JWTAuth
+ * @author       XWP
+ * @license      GPL-2.0+
+ *
+ * @wordpress-plugin
+ * Plugin Name:       JWT Auth
+ * Plugin URI:        https://github.com/WP-API/jwt-auth
+ * Description:       Feature plugin to bring JSON Web Token REST API authentication to Core
+ * Version:           0.1
+ * Author:            XWP, and contributors
+ * Author URI:        https://github.com/WP-API/jwt-auth/graphs/contributors
+ * Text Domain:       jwt-auth
+ * License:           GPL-2.0+
+ * License URI:       http://www.gnu.org/licenses/gpl-2.0.txt
+ * GitHub Plugin URI: https://github.com/WP-API/jwt-auth
+ * Requires PHP:      5.3.0
+ * Requires WP:       4.4.0
+ */
+
+define( 'JWT_AUTH_PLUGIN_DIR', dirname( __FILE__ ) );
+define( 'JWT_AUTH_PLUGIN_URL', plugin_dir_url( __FILE__ ) );
+define( 'JWT_AUTH_VERSION', '0.1' );
+
+/**
+ * Requires running PHP 5.3 or above.
+ *
+ * @since 0.1
+ * @codeCoverageIgnore
+ */
+function jwt_auth_version_check() {
+
+	if ( version_compare( PHP_VERSION, '5.3', '<' ) ) {
+		deactivate_plugins( plugin_basename( __FILE__ ) );
+		wp_die( esc_html__( 'The JWT Auth plugin requires PHP Version 5.3 or above.', 'jwt-auth' ) );
+	}
+}
+add_action( 'admin_init', 'jwt_auth_version_check' );
+
+/**
+ * Load the JWT Auth plugin.
+ *
+ * @since 0.1
+ */
+function jwt_auth_loader() {
+
+	/** JWT Classes */
+	foreach ( glob( JWT_AUTH_PLUGIN_DIR . '/wp-includes/rest-api/auth/jwt/*.php' ) as $filename ) {
+		require_once $filename;
+	}
+
+	/** WP_REST_Token Class */
+	require_once JWT_AUTH_PLUGIN_DIR . '/wp-includes/rest-api/auth/class-wp-rest-token.php';
```

Comment:
PHPDoc question: is `/** */` the appropriate syntax here? I'd expect that `/**` would be used within that file, and this would be `//`, but i'm less familiar with PHPDoc than JSDoc and others.